### ISSUE TITLE
Accepting PRs on TypeScript rewrite?

### ISSUE THREAD

**Issue by paarthenon** [chancejs/chancejs#469]
Hey there,

I noticed the 2.0 branch of the project is built, but pretty bare at the moment. Are you interested in some help on that branch? And do you have any guidelines for it? I'm pretty much wondering how interested you are in maintaining the 1.0 API (in terms of feature parity and/or structure) and how you'd like to split the functionality between the packages in the lerna repo.

**Comment by victorquinn** [comment]
Yes gladly! Apologies for my delay in response here, didn't see this earlier.

Mostly I've gotten it to a point of needing a lot of rote migration and haven't had a ton of time myself so it kind of stalled unfortunately. But the base framework is built up to where it should be fairly straightforward for others to jump in and help.

I put most of my thoughts here on #438 and would love small atomic PRs for each generator and can give feedback there. 

But mostly my hope was to have the clean code-splitting, TS, and the main change is in the initialization. Otherwise I hoped to keep most of the generators with a similar API to what they have today.

**Comment by victorquinn** [comment]
Just added this file to the 2.0 branch on contributing which should help https://github.com/chancejs/chancejs/blob/2.0/CONTRIBUTING.md

**Comment by paarthenon** [comment]
Hey Victor I just added this, let me know if that is up to snuff.

https://github.com/chancejs/chancejs/pull/476
